extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -270,6 +271,13 @@ public void onConnectionLost(HueBridge bridge) {
     public void onConnectionResumed(HueBridge bridge) {
         logger.debug("Bridge connection resumed. Updating thing status to ONLINE.");
         updateStatus(ThingStatus.ONLINE);
+        // now also re-initialize all light handlers
+        for (Thing thing : getThing().getThings()) {
+            ThingHandler handler = thing.getHandler();
+            if (handler != null) {
+                handler.initialize();
```

Comment:
The initialize() method imho must never be called by bindings itself - this is a method that is called by the framework only. If this needs to be called, this is a bad smell and asks for a refactoring of either the binding or the framework.